ui/app/component/rds/admin/RdsAdminController.js

```diff
@@ -59,6 +59,10 @@ class RdsAdminController extends GatekeeperAdminController{
                         filterFn: vm.filterGk
                     }]
             },
+            export: {
+                filename: 'rds-users',
```

Comment:
Is it possible to name the file based on the instance that was selected?

You could probably update this value from the getUsers method in the controller to set the filename to be

`export.filename="${row.instanceId}_rds-users"`

every time an instance is selected